src/main/java/com/ldtteam/structurize/management/Structures.java

```diff
@@ -106,6 +107,12 @@ private Structures()
      */
     public static void init()
     {
+        md5Map.clear();
```

Comment:
this looks weird - elaborate why is this needed

Reply:
In testing, I noticed that deleting blueprint files and reloading the world, or switching between different worlds with different sets of blueprints in the same MC client session, did not properly reset the list of available blueprints and it got very confused.  This fixes that.

Reply:
It was also very noticeable when switching between different worlds with different section configs.